### PR TITLE
refactor: navigate to internal collection pages for links

### PR DESCRIPTION
**Pull request by patricio0312rev**
# [[Featured Collections] Navigate to internal collection url](https://app.clickup.com/t/862kcb9e3) ~& [[Galleries] Route changes when viewing NFT card](https://app.clickup.com/t/862kcb3c0)~

## Summary

- The title of the collection on each card redirects to the collection details page.
- Each featured collection icon in the carousel we display at the bottom of a gallery will now redirect to the internal collections detail page.

## Steps to reproduce
1. Run the app in `test_2e2` mode.
2. Go to `Galleries` and select any gallery.
3. ~Click on a card and expect to be redirected to the internal NFT details page.~
4. Click on the collection's name when hovering over the card and expect to be redirected to the internal collection details page.
5. On the same page, scroll down and click on any of the featured collections displayed in the banner. Expect to be redirected to the internal collections details page.

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: resources/js/Components/Galleries/GalleryPage/GalleryNftsNft.tsx [hunk 1]
```diff
@@ -80,14 +79,13 @@ export const GalleryNftsNft = ({ nft, isSelected, onClick }: Properties): JSX.El
                             />
                         </div>
                     )}
-                    <Link
-                        external
-                        href={nft.collectionWebsite}
+                    <div
```
Comment (ItsANameToo) [inline resources/js/Components/Galleries/GalleryPage/GalleryNftsNft.tsx:82]: if we make a `div` clickable, it's best to add `role="button"` for accessibility. Although, why isn't this still a `Link` where you remove the `external` attribute as it's no longer an external link?

File: resources/js/Components/Galleries/GalleryPage/GalleryNftsNft.tsx [hunk 2]
```diff
@@ -46,28 +44,29 @@ export const GalleryNftsNft = ({ nft, isSelected, onClick }: Properties): JSX.El
...
+            route("collections.view", {
+                slug: nft.collectionSlug,
+            }),
+        );
     };
 
     return (
         <GalleryCard
-            isSelected={isSelected}
+            isSelected={false}
```
Comment (ItsANameToo) [inline resources/js/Components/Galleries/GalleryPage/GalleryNftsNft.tsx:66]: is this property still used anywhere or now obsolete?

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

**Comment by ItsANameToo** [comment]
I removed the redirect to the NFT page on click, description has been updated to reflect that

**Review by samharperpittam — APPROVED** [review]
Collection navigation works great. Will provide an update for NFT navigation once i've discussed with others,

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**